### PR TITLE
Add uBlock filters – Badware risks

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/197

### CODE REVIEW

File: filters/security/filter_50_UblockBadwareRisks/metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "For sites documented to put users at risk of installing adware/crapware/malware, having login credentials stolen, etc. The purpose is to at least ensure a user is warned of the risks ahead.",
+    "timeAdded": 1695145147610,
+    "homepage": "https://github.com/uBlockOrigin/uAssets",
+    "expires": "4 days",
+    "displayNumber": 4,
+    "environment": "dev",
+    "tags": [
+      "purpose:security"
+    ]
+  }
```
Comment (slavaleleka) [inline filters/security/filter_50_UblockBadwareRisks/metadata.json:14]:
```suggestion
}
```

File: filters/security/filter_50_UblockBadwareRisks/metadata.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "filterId": "ublock_badware_risks",
```
Comment (slavaleleka) [inline filters/security/filter_50_UblockBadwareRisks/metadata.json:2]: нужно выровнять табы — тут 1 отступ это 4 пробела, а ниже по 2 пробела

**Comment by ameshkov** [comment]
Please check the result of HostlistCompiler work for this filter list, I am not entirely sure that we support everything that's required.

File: filters/security/filter_50_UblockBadwareRisks/metadata.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "uBlock₀ filters – Badware risks",
+  "description": "For sites documented to put users at risk of installing adware/crapware/malware, having login credentials stolen, etc. The purpose is to at least ensure a user is warned of the risks ahead.",
+  "timeAdded": 1695145147610,
+  "homepage": "https://github.com/uBlockOrigin/uAssets",
+  "expires": "4 days",
+  "displayNumber": 4,
+  "environment": "dev",
+  "tags": [
+      "purpose:security"
+    ]
```
Comment (slavaleleka) [inline filters/security/filter_50_UblockBadwareRisks/metadata.json:13]: tab again

**Comment by Alex-302** [comment]
@Sergey-Lyapin Did you check https://github.com/AdguardTeam/HostlistsRegistry/pull/349#issuecomment-1726225690 ?

**Comment by Sergey-Lyapin** [comment]
@ameshkov С текущей конфигурацией вот такой файл на выходе получается:
[output.txt](https://github.com/AdguardTeam/HostlistsRegistry/files/12684065/output.txt)

**Comment by ameshkov** [comment]
Looks good actually

**Review by ameshkov — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Comment by Sergey-Lyapin** [comment]
@slavaleleka 
Please approve if it's ok.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**